### DISCUSSION TITLE
Request limits of serverless functions on vercel.

### DISCUSSION THREAD

**Discussion by kcubeterm** [vercel/vercel#6315] [General]
Like Cloudflare worker free tier offer 100k request per day, AWS Lamda offer 1 Million request per months. What's the limits  on Vercel tier ? Did't find any docs that's why asking.

**Comment by fu-sen** [comment]
Here is the limit information:
https://vercel.com/docs/platform/limits

If you haven't paid Vercel outside of your custom domain, it's a Hobby plan.
